packages/api/curl-scripts/README.md

```diff
@@ -0,0 +1 @@
+For testing 
```

Comment:
Do you think it might be worth adding some example cURL requests?

Reply:
I think you could even just point to `packages/api/curl-scripts/setup-applications.sh` as that has them!

Reply:
Its documented in the swagger-ui really and you can generate curl request from that.

Reply:
Fair enough, will resolve then